backend/src/main/java/hu/flowacademy/worksheet/controller/PartnerController.java

```diff
@@ -15,6 +17,7 @@
 import java.util.Optional;
 
 @RequiredArgsConstructor
+@Slf4j
```

Comment:
Én lennék a legboldogabb, ha logolnátok néha, de ez így itt felesleges. de no need to be upset! (https://www.youtube.com/watch?v=eY52Zsg-KVI)